### DISCUSSION TITLE
Welcome to Greaseweazle Discussions!

### DISCUSSION THREAD

**Discussion by keirf** [keirf/greaseweazle#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ourIThome** [comment]
I'm a big fan of greaseweazel.

**Reply by AlKossow** [reply]
Hi, https://github.com/schlae has made his own version of the F7 with buffering and 8" floppy drive support.
I'm a little surprised he didn't put it on github 
![esgw](https://user-images.githubusercontent.com/9269979/102727355-13f41880-42da-11eb-82f2-56b358388438.JPG)

**Reply by keirf** [reply]
Interesting. I wonder if it presents as an "F7 Plus" board?

**Reply by gps-au** [reply]
I love the idea of not having to put a 34/50 converter in...

**Comment by wohali** [comment]
Hi there! I'm a hardware engineer and software developer. I recently did a run of GW F7 1.1s at cost to the community, and am looking forward to helping with future hardware revs.

I'd also like to help with adding more formats to the system, and would love to help out with some sort of streaming connection with the Disk-Utilities library so that reads of diskettes can be validated as it's being dumped, for archival purposes.

**Reply by keirf** [reply]
That's great, there are a few hardware models now, but always happy to discuss further extensions.

Linking in Disk-Utilities is an interesting possibility. I'm currently working on a C extension just to get performance up, but importing the Disk-Utilities analysers would be great, as there are a lot of Amiga ones there. Realistically they won't all be native to Greaseweazle any other way any time soon. Though I am adding analysers slowly (IBM MFM in progress, but finally I now need a config file format so that track layouts can be specified).

**Comment by andrewbird** [comment]
Hi I don't have a Greaseweazel yet, my proposed use is with PC formats, but I don't have any experience with surface mount assembly. The `F7 Lightning Plus`  looks ideal, where should I get one from as I don't use Facebook?

**Reply by keirf** [reply]
You can email me. I should put up a "sales" post here with the prices.

**Reply by keirf** [reply]
I've now made a sales discussion with all details.

**Reply by andrewbird** [reply]
Cool thanks.

**Reply by wohali** [reply]
Hey @keirf , did you remove the sales post?

**Reply by keirf** [reply]
Yes I'm mainly selling F7 v3s right now. I can be emailed about them. They're £25 plus postage.

**Comment by mtps37** [comment]
Hello.

setup.sh is missing in Linux version v0.22 of Greaseweazle software:
[https://github.com/keirf/Greaseweazle/wiki/Software-Installation#linux](url)

Edit: I've seen it's corrected for 0.24 version.
